R/oarg_decode.R

```diff
@@ -19,10 +19,6 @@
 #' @param records_path directory for the xml files
 #' @param verbose print some information on what is going on
 #'
-#' @importFrom progress progress_bar
-#' @importFrom purrr pmap
-#' @importFrom base64enc base64decode
-#' @importFrom tibble tibble
```

Comment:
these are redundant; if we always call `foo::bar()`, then we don't need to pollute our namespaces.
I think `@importFrom` makes most sense for those packages (very few) where we as pkg devs can't be bothered always explicating `foo::bar()`.
Whenever possible the tidyverse style guide advises being explicit.